src/Storages/System/StorageSystemMetrics.cpp

```diff
@@ -34,6 +38,53 @@ void StorageSystemMetrics::fillData(MutableColumns & res_columns, ContextPtr, co
         res_columns[0]->insert(CurrentMetrics::getName(CurrentMetrics::Metric(i)));
         res_columns[1]->insert(value);
         res_columns[2]->insert(CurrentMetrics::getDocumentation(CurrentMetrics::Metric(i)));
+        res_columns[3]->insertDefault();
+    }
+
+    const auto & factory = Histogram::Factory::instance();
```

Comment:
MetricLog (`src/Interpreters/MetricLog.cpp`) should also display this, right?

Reply:
Well, adding histogram metrics to the metric log will be quite clumsy because, for each histogram metric, we add (number of unique label combinations) * (number of buckets + 1 + 2) metrics. Besides, they are exported to Prometheus, where exploring their history is far more convenient.